scripts/entrypoint-scheduler.sh

```
@@ -1,27 +1,2 @@
-echo "════════════════════════════════════════════════════════════════"
-echo ""
-echo "Configuration:"
-echo "  Check Interval: ${SCHEDULER_CHECK_INTERVAL:-60}s"
-echo "  Log Level: ${SCHEDULER_LOG_LEVEL:-INFO}"
-echo ""
-echo "════════════════════════════════════════════════════════════════"
-echo ""
-
-run_scheduler --check-interval="${SCHEDULER_CHECK_INTERVAL:-60}" \
-    --log-level="${SCHEDULER_LOG_LEVEL:-INFO}"
+# Entrypoint for scheduler container (scheduler via supervisord)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)